backend/src/main/java/com/ambacam/model/Pays.java

```diff
@@ -0,0 +1,69 @@
+package com.ambacam.model;
+
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Entity
+public class Pays implements Serializable {
```

Comment:
please generate the serialVersionUID (put your mouse on the name of the class and then choose "add generate serial version ID")

Reply:
This is not possible on intellij

Reply:
Have you tried this?
put your mouse on the name of the class and then choose "add generate serial version ID"

Reply:
If it does not work. Here is a link that explains you how to activate and generate it:
https://netlicensing.io/blog/2012/12/25/intellij-idea-how-to-generate-serialversionuid/